src/plugins/ui_stateful/clipboard.ts

```diff
@@ -52,19 +52,31 @@ export class ClipboardPlugin extends UIPlugin {
 
   allowDispatch(cmd: LocalCommand): CommandResult {
     switch (cmd.type) {
+      case "COPY":
+        this.allowPaste = true;
+        break;
```

Comment:
We should't change the state of the plugin in the `allowDispatch`. It doesn't change much ATM, but we could imagine another plugin that have an allowDispatch on `COPY`. We could then have the situation:

- ClipboardPlugin allowDispatch COPY, set `allowPaste` to true
- Random plugin block the COPY in its allowDispatch.
- In this case the state of the ClipboardPlugin is wrong, because allowPatch = true, but we didn't actually copy anything.

You did well adding the `CommandResult.RepeatitivePasteAfterCut` in the allowDispatch. But the changes on the allowPaste should stay in the `handle`.